packages/w3c-vc/README.md

```
@@ -429,6 +431,147 @@ const signedDocument = { ...document, proof: { /* proof object */ } };
+
+console.log(result.verified);          // true only if the holder proof + all credentials verify + not expired
+console.log(result.presentationResult); // outcome of the holder proof (undefined if unsigned)
+console.log(result.credentialResults);  // per-credential outcomes (with credentialIndex)
+```
+
+> **`challenge`/`domain` are optional (per the W3C Data Integrity spec).** Provide a
+> `challenge` (Route 3) for **single-use** anti-replay — this needs a **server** to issue a
+> fresh nonce and track/consume it. Omit it (Route 2) for a signed-but-replayable proof.
+> `verifyPresentation` requires you to pass the `challenge` **it** issued — the value in the
+> proof is never trusted (that would allow replay).
+
```

markdownlint-cli2 (0.23.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)